docs/userguide/networking/dockernetworks.md

```diff
@@ -404,16 +404,26 @@ provision the hosts are with Docker Machine.
 
 ![Engine on each host](images/engine_on_net.png)
 
+You should open the following ports between each of your hosts.
+
+| Protocol | Port | Description           |
+|----------|------|-----------------------|
+| udp      | 4789 | Data plane (VXLAN)    |
+| tcp/udp  | 7946 | Control plane         |
+
+Your key-value store service may require additional ports. 
+Check your vendor's documentation and open any required ports.
+
 Once you have several machines provisioned, you can use Docker Swarm to quickly
 form them into a swarm which includes a discovery service as well.
 
 To create an overlay network, you configure options on  the `daemon` on each
 Docker Engine for use with `overlay` network. There are two options to set:
 
-| Option                           | Description                                               |
-|----------------------------------|-----------------------------------------------------------|
-| `--cluster-store=PROVIDER://URL` | Describes the location of the KV service.               |
-| `--cluster-advertise=HOST_IP`    | Advertises containers created by the HOST on the network. |
+| Option                                        | Description                                                 |
+|-----------------------------------------------|-------------------------------------------------------------|
+| `--cluster-store=PROVIDER://URL`              | Describes the location of the KV service.                   |
+| `--cluster-advertise=HOST_IP|HOST_IFACE:PORT` | The IP address or interface of the HOST used for clustering |
```

Comment:
FYI - the `|` in `--cluster-advertise=HOST_IP|HOST_IFACE:PORT` gets interpreted as a cell break, and messes up rendering in the docs site (https://docs.docker.com/engine/userguide/networking/dockernetworks/)...